apps/indexer/bin/webhook-event-delivery-worker.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node --experimental-transform-types
+
+import * as Sentry from "@sentry/node";
+import "./init.ts";
+import { env } from "../src/env.ts";
+import { webhookEventDeliveryService } from "../src/services/index.ts";
```

Comment:
this index.ts file gonna start a bunch of services that is unrelated to current process, even it is not problem now, this feels a bit risky to me, someone later might accidentally put a service that should be started twice concurrently there.

Reply:
moved the service instantiation directly to the workers since there is only one service necessary